package.json

```diff
@@ -105,6 +106,7 @@
     "remix-auth-github": "^1.7.0",
     "remix-utils": "^8.1.0",
     "set-cookie-parser": "^2.7.1",
+    "sharp": "^0.33.5",
```

Comment:
Why is this a pure dependency of OpenImg and not just a built-in dependency? Why do I need to install it myself?

Reply:
This decision was made based on a gut feeling. Sharp uses [libvips](https://github.com/libvips/libvips) under the hood, and bundling binaries can get tricky. I figured it was better to let you control how to bundle sharp and choose the version you need. I believe Next.js takes a similar approach by requiring you to install sharp yourself (see [here](https://nextjs.org/docs/messages/sharp-missing-in-production)). But happy to change my mind if you think it's better otherwise!

Reply:
Sounds good to me!